### PR TITLE
Revised Blog Content for Kanvas Catalog

### LINKED ISSUE #6066: Add guide on how to contribute to the Catalog
### Current Behavior

https://layer5.io/blog/kanvas/what-is-the-kanvas-catalog
image
`How can you contribute to the Catalog?`
### Desired Situation

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by TraXIcoN**
**Description**

This PR fixes #6066 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c71203fbcea7b0ce8ff67e06544991500872de96 at: https://672d310d38981963ab12992b--layer5.netlify.app

File: src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/post.mdx [hunk 1]
```diff
@@ -17,32 +17,114 @@ resource: true
...
 - Accelerated Development: Leverage existing patterns to jumpstart your projects and avoid reinventing the wheel.
-- Community Knowledge: Benefit from the collective experience of the Layer5 community and industry best practices.  
+- Community Knowledge: Benefit from the collective experience of the Layer5 community and industry best practices.
 - Standardization: Promote consistency and reduce errors by using predefined configurations.
 - Collaboration: Share your own designs and contribute to the growing collection of patterns.
 
 ### How can you contribute to the Catalog?
- 
 
+Contributing to the Meshery Catalog is a great way to share your cloud-native configurations with the community. By submitting your models and patterns, you not only showcase your work but also help others benefit from your innovations. This guide outlines how to use `mesheryctl` for uploading your contributions efficiently.
```
Comment (leecalcote) [inline src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/post.mdx:54]: Are we talking about the Meshery Catalog? 

Double check the title of this post.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/post.mdx [hunk 2]
```diff
@@ -17,32 +17,114 @@ resource: true
...
+**Components in Meshery**:  
+Components are entities in the Meshery ecosystem that expose platform capabilities. They can be:
+
+- Registered, created, and used by users and operators
+- Defined through definitions, instances, and metadata
+- Identified by their kind, apiVersion, and model.name attributes (duplicates share these attributes)
+
+<img src={MesheryModelsBreakdown}></img>
+
+#### Contributing with mesheryctl
```
Comment (leecalcote) [inline src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/post.mdx:76]: How do Kanvas users use the UI to create a new model?

Comment (leecalcote) [reply src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/post.mdx:76]: I see the section below.

File: src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/meshery-models-breakdown.png
Comment (leecalcote) [inline src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/meshery-models-breakdown.png:1]: Please also remove the black box in the lower left corner with out-dated statistics.

Comment (leecalcote) [reply src/collections/blog/2024/11-05-what-is-the-kanvas-catalog/meshery-models-breakdown.png:1]: Also, export to SVG, not PNG, please.

**Pull request closed without merging**